Description: fix "Unescaped left brace" warning, which is fatal in 5.25.x
Origin: vendor
Bug: https://rt.cpan.org/Public/Bug/Display.html?id=115103
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=826463
Forwarded: https://rt.cpan.org/Public/Bug/Display.html?id=115103
Author: gregor herrmann <gregoa@debian.org>
Last-Update: 2017-05-20

--- a/build/Wx/build/Utils.pm
+++ b/build/Wx/build/Utils.pm
@@ -107,7 +107,7 @@
     m/^\s*INCLUDE:\s+(.*)$/ and $file = $1 and $arr = \@xsinclude;
     m/^\s*INCLUDE_COMMAND:\s+.*\s(\S+\.(?:xsp?|h))\s*/ and $file = $1 and
       $arr = \@xsinclude;
-    m/^\s*\%include{([^}]+)}\s*;\s*$/ and $file = $1 and $arr = \@xsinclude;
+    m/^\s*\%include\{([^}]+)}\s*;\s*$/ and $file = $1 and $arr = \@xsinclude;
 
     if( defined $file ) {
       $file = catfile( split '/', $file );
